cmd/noms/noms.go

```diff
@@ -44,49 +29,70 @@ var actions = []string{
 	"nomming on",
 }
 
-func usageString() string {
-	i := rand.New(rand.NewSource(time.Now().UnixNano())).Intn(len(actions))
-	return fmt.Sprintf(`Noms is a tool for %s Noms data.`, actions[i])
+// CommandHandler - a callback passed to commands
```

Comment:
This comment doesn't look correct... I think this type actually is the type that command handlers must implement. You probably don't even need a comment since that's what the name says.